ctc_lib/ctc_lib.go

```diff
@@ -41,17 +51,41 @@ func GetExitOnError() bool {
 }
 
 func CommandExit(err error) {
-	if err != nil && exitOnError {
-		fmt.Println(err)
-		os.Exit(1)
+	if err != nil {
+		logging.Out.Error(err)
```

Comment:
I must have missed this in a previous PR, but can you explain `logging.Out`? is this to support different logging destinations?

Reply:
Yes you are right. logging.Out is just a convenience object which can be used to redirect output to stdout when needed. 
One, usecase is 
Before Exiting, we need to print err to StdOut.
If a command line developer want to log to stdout, he can use this object